docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/CgovCoreTools.php

```diff
@@ -568,7 +567,7 @@ public function getCloudEnvironment() {
 
     // Check if site_env matches a prod environment name...
     if (in_array($site_env, self::PROD_AH_SITE_ENVS) ||
-      preg_match('/^.*(www-prod-acsf|www-cms).*$/', $site_env)) {
```

Comment:
Hey @jfrank-nih -- I don't think this is something that should have been changed... But also, why is only this here?